javascript/sdk/test/sdk.test.ts

```diff
@@ -114,7 +114,7 @@ describe("SDK Test", () => {
     });
 
     it("createTest should not throw an error", async () => {
-      await service.build.createTest(testId, testName, testUnit);
+      await service.build.createTest(testName, testUnit, undefined, testId);
```

Comment:
is this 'undefined' really how JS sdks work? we can't name/default the params?

Reply:
@stephanwampouille i do prefer the CreateTestParams btw. You can see i used a similar pattern other functions